frontend/src/types/status.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-shadow */
+export enum Status {
```

Comment:
Generally better to use literal types than adding an enum, to make comparing Python types and JS types easier. Also, I don't believe there is a need for a "pending" order type, as they should go through (or not) in the same request.

Reply:
I think you've fixed the typing here a while ago and this file is just as-yet-undeleted zombie file :D
"Pending" is there as I'm not sure how the api works - we can delete that once we've got it attached?

Reply:
Got it :)